printer_mods/MathematicalPotato/v0.1_belted_z_drive/README.md

```diff
@@ -12,7 +14,10 @@ The belt clamp/bed mount part accounts for the 3.2mm shift forward for the bed o
 * Designed the bottom mount to utilize V0.0 M3 nuts already in the 1515 extrusions. For a V0.1 you'll need to add one extra M3 nut in the back of each Z rail extrusion and two in the rear of the rear bottom extrusion.
 * Optional bottom panel filler can also be used for this version.
 
-![Zbelt-1](Images/ZBelt-ISO-02.png)
+
+* V0-zbelt-BEDMOUNT_MODIFIED_V0.1 has built in supports that need to be removed.
+![Zbelt-1](Images/ZBelt-Support-01.PNG)
```

Comment:
This file you named with lowercase `.png`, but reference it here with uppercase `.PNG`, so it's a broken image.

Reply:
I resubmitted the README with a link to FPVGeek's design, but all of the image links show the correct formatting on my end.  Hopefully this resubmission will go through properly.
[README.md](https://github.com/VoronDesign/VoronUsers/files/6704203/README.md)

Reply:
This image link has correct formatting, but that file doesn't exist.  For just this image, you need to lowercase `.png`, because that's what you named that file:

<img width="275" alt="Screen Shot 2021-06-23 at 17 21 41" src="https://user-images.githubusercontent.com/69389593/123169715-a327e780-d447-11eb-825f-8955a8f004b1.png">

This is what the image looks like right now:

<img width="805" alt="Screen Shot 2021-06-23 at 17 20 15" src="https://user-images.githubusercontent.com/69389593/123169757-b0dd6d00-d447-11eb-8d26-a366a9887f14.png">

Reply:
Ok, should be good to go now.  I also realized I hadn't put the step file into the CAD folder so I've now added that as well